package.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
Move this file to front-end folder. 
All the project dependencies for front-end should live on front-end folder